.gitignore

```diff
@@ -1,3 +1,4 @@
 **/target
 src/template_files.rs
 template/Cargo.lock
+/.vscode
```

Comment:
but maybe not in template

Reply:
I specifically checked that it does not ignore the template

Reply:
![image](https://github.com/user-attachments/assets/849813f1-377b-4baa-b7de-13393033c01c)